platform/windows/display_server_windows.cpp

```diff
@@ -227,7 +227,6 @@ Error DisplayServerWindows::file_dialog_show(const String &p_title, const String
 	for (const String &E : p_filters) {
 		Vector<String> tokens = E.split(";");
 		if (tokens.size() == 2) {
-			filter_exts.push_back(tokens[0].strip_edges().utf16());
```

Comment:
My bad missed it was an edited line, it should be restored

Reply:
Got it. I switched to my home computer and didn't notice this.